addons/stock_dropshipping/tests/test_dropship.py

```diff
@@ -297,3 +297,53 @@ def test_correct_vendor_dropship(self):
         self.assertTrue(purchase, "an RFQ should have been created by the scheduler")
         self.assertTrue((purchase.date_planned - purchase.date_order).days == 10, "The first supplier has a delay of 10 days")
         self.assertTrue(purchase.amount_untaxed == 8, "The price should be 4 * 2")
+
+
+@tagged('post_install', '-at_install')
+class TestDropshipPostInstall(common.TransactionCase):
+
+    def test_dropshipping_tracked_product(self):
+        supplier, customer = self.env['res.partner'].create([
+            {'name': 'Vendor Man'},
+            {'name': 'Customer Man'},
+        ])
+        product_lot = self.env['product.product'].create({
+            'name': "Serial product",
+            'tracking': 'serial',
+            'standard_price': 20,
+            'invoice_policy': 'delivery',
+            'seller_ids': [Command.create({
+                'partner_id': supplier.id,
+            })],
+            'route_ids': [Command.link(self.ref('stock_dropshipping.route_drop_shipping'))]
+        })
+        product_lot.tracking = 'serial'
+        product_lot.categ_id.property_cost_method = 'standard'
+        product_lot.standard_price = 20
```

Comment:
```suggestion
        product_lot.categ_id.property_cost_method = 'standard'
```
`tracking` and `standard_price` are already set in the `create()`.